roles/pulp3-content/meta/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+galaxy_info:
+  author: Pulp
+  description: Pulp content app
+  company: Red Hat
+  min_ansible_version: 2.4
+  platforms:
+    - name: Fedora
+      versions:
+        - 28
```

Comment:
Since our tests use f27 and centos, I think we should include them too.

Reply:
I realize that you probably did it this way to mirror the workers role, which is also missing those platforms. Would you mind changing that as well while we are thinking about it?

Reply:
Fedora 27 is end of life, IMO we should disinclude it both here and with the tests, and use F28+

Reply:
Good call. Filed: https://pulp.plan.io/issues/4334